tests/conftest.py

```diff
@@ -31,7 +31,7 @@ def fixture_load_parsl() -> None:
 
 
 # note: we use name here to avoid pylint flagging W0621
-@pytest.fixture(name="get_tempdir", scope="session")
+@pytest.fixture(name="get_tempdir", scope="function")
```

Comment:
Kind of a silly thing, but since `scope="function"` is the default value, maybe it makes sense to leave it out?

Reply:
Good point, I'll remove to defer to the default here.